### ISSUE TITLE
Expose routerReducer so it works with @rematch/core

### ISSUE THREAD

**Issue by linonetwo** [supasate/connected-react-router#79]
Regarding https://github.com/rematch/rematch/issues/409

@rematch/core is exposing a field to put rootReducers, but it can only receive a reducer, but not a function returning a reducer.  https://github.com/rematch/rematch/blob/master/docs/reduxApi.md#rootreducers

Can `connected-react-router` expose this reducer directly without high order function?
https://github.com/supasate/connected-react-router/blob/master/src/reducer.js#L15

**Comment by dantonijevic** [comment]
Hey @linonetwo, I had a similar problem while using [redux-sagas-injector](https://github.com/GuillaumeCisco/redux-sagas-injector). I needed the reducer exposed too, so I came up with a hack where you can get the reducer without exposing it from the library.

```
// pass the empty function for the rootReducer 
let routerReducer = connectRouter(history)(() => {}); 

// then use routerReducer as regular reducer in your reducers map with an extra line of code
let allMyReducers = {
...allMyReducers,
router: (state, action) => { let result = rootReducer(state, action); return state.router }
}
```

Let me know if it works for you, and please, let me know if I am missing something with this approach. Thanks!

**Comment by linonetwo** [comment]
Sorry @dantonijevic , I'm not using `connected-react-router` and rematch anymore. I've switched to Apollo-GraphQL for more convenient data fetching...

**Comment by bdwain** [comment]
@linonetwo can you reopen this issue please? it's still relevant I think even though you're not using it anymore.

cc @supasate This would be really helpful to use with [redux-loop](https://github.com/redux-loop/redux-loop)

**Comment by bdwain** [comment]
bump @supasate any thoughts on this?

**Comment by supasate** [comment]
PR #140 is addressing this issue.

**Comment by bdwain** [comment]
cool thanks!

**Comment by supasate** [comment]
Closing as already fixed in #150.
